mlflow/pytorch/__init__.py

```diff
@@ -325,7 +326,10 @@ def save_model(
         f.write(pickle_module.__name__)
     # Save pytorch model
     model_path = os.path.join(model_data_path, _SERIALIZED_TORCH_MODEL_FILE_NAME)
-    torch.save(pytorch_model, model_path, pickle_module=pickle_module, **kwargs)
+    if isinstance(pytorch_model, torch.jit.ScriptModule):
```

Comment:
Could we update the docstrings of `log_model` and `load_model` to mention that you can now also log a TorchScript model (i.e. the output of `torch.jit.trace` or `torch.jit.script`)?

Reply:
It'd also be good to update the documentation of ``mlflow.pytorch.load_model`` to clarify that we load back either an eager model (subclass of `torch.nn.Module`) or a TorchScripted model, depending on whether an eager or TorchScripted model was originally logged.

Reply:
Updated the doc strings by mentioning the support of scripted model. In the example section, added the code to log and save model as well

Reply:
We will raise a separate PR for updating documentation of `mlflow.pytorch`